### ISSUE TITLE
Update Copyrights

### ISSUE THREAD

**Issue by gmkurtzer** [apptainer/singularity#6136]
We should update the copyrights on the LICENSE.md, COPYRIGHT.md, and all source files to include:

```
Copyright (c) 2020-2021, HPCng / Singularity Community. All rights reserved.
```

**Comment by DrDaveD** [comment]
Does it make sense for a community to have a copyright?  Doesn't it need to be some individual or organization?

**Comment by gmkurtzer** [comment]
It could/should be HPCng

**Comment by PaulCharlton** [comment]
@gmkurtzer -- wondering if there is legal precedent one way or the other.   a community can be considered a "joint venture" which is generally recognized at law in the same manner as any other non-person entity.  what is less clear is who would have the authority to sign contracts involving specific rights

**Comment by DrDaveD** [comment]
Fixed in apptainer/apptainer#1
